Changes

```diff
@@ -1,13 +1,24 @@
 Release history for Zonemaster component Zonemaster-LDNS
 
+
+3.0.0 2019-11-15
+
+ [API change]
```

Comment:
What's the thought behind the change in format? Is it to comply with [CPAN::Changes::Spec](https://metacpan.org/pod/distribution/CPAN-Changes/lib/CPAN/Changes/Spec.pod)? Maybe we should mention that in the Versions and Releases document?

Reply:
Good idea to add the reference to documentation. Will look at that.